KrishnaSharma206.md

```diff
@@ -0,0 +1,33 @@
+---
+name: "@KrishnaSharma206"
+project: "An AI powered Robot"
+---
+
+# Object Recognition Warehouse Robot
+
+## Summary
+
+A robot that recohnizes certain shapes and navigates it's way accordingly, I am going to build this robot because I liked the idea of the amazon warehouse robot and the projects made by a guy named James Bruton, and I really wanna learn about object recognition through this project. I already have some experience making robots using arduino, so it will be something new for me.
+
+## Plan
+
+I will be using the jetson nano as the brain of this project, the raspberry pi wont be enough for this and the jetson nano will be a better alternative
+, I will be using a camera for the detection and I am planning to add more sensors and actuators to it in the future.
+
+The Steps:
+1. Learn how to use the jetson nano
+2. Connect my laptop to the jetson nano using a vnc
+3. Train the AI model on the jetson nano using basic shapes for easier recognition
+4. Build the robot, then program the robot using python
+5. Test the robot.
+
+## Budget
+
+| Product                | Supplier/Link                                               |  Cost   |
+| ---------------        | ----------------------------------------------------------- | ------- |
+| Nvidia Jetson nano     | https://www.arrow.com/en/products/945-13450-0000-100/nvidia | $195.00 |
+| Rover5 chassis         | https://www.rhydolabz.com/robotics-robotic-kits-c-155_161/dagu-rover-5-2wd-tracked-chassis-without-encoder-p-1258.html | $39 |
+| Rover5 expansion plate | https://www.pololu.com/product/1530                         | $16.00  |
+| Total                  |                                                             | $250    |
+
+Note: I will be buying rest of the parts required by myself, these are the parts that are not available in my country and also are too expensive for me to buy.
```

Comment:
Could you list the rest of the parts in your `budget` but mention that you'll purchase them on your own?

Reply:
Ok